docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -1221,17 +1213,158 @@ paths:
             $ref: '#/definitions/Problem'
         '409':
           description: |
-            Conflict. There are no empty slots for this subscriptions. The amount of consumers for this subscription
+            Conflict. There are several possible reasond for receiving this status code:
+
+            - There are no empty slots for this subscriptions. The amount of consumers for this subscription
+            already equals the maximal value - the total amount of this subscription partitions.
```

Comment:
maximum

Reply:
total number of partitions in this subscription

Reply:
fixed